torch/_tensor_str.py

```diff
@@ -364,12 +364,13 @@ def _str_intern(inp):
         suffixes.append('size=' + str(tuple(self.shape)))
         if not has_default_dtype:
             suffixes.append('dtype=' + str(self.dtype))
-        suffixes.append('quantization_scheme=' + str(self.qscheme()))
```

Comment:
this code sounds right, what's the eng cost to fix the underlying issue so this code prints the expected thing?

Reply:
One way to do this would be accepting the qscheme as an input to `quantize_per_tensor`/`quantize_per_channel` and store that information with the tensor. However, internally we create a Quantizer based on the args passed in and currently we only have AffineQuantizer defined. I see that we did have SymmetricQuantizer in the past but it was removed in https://github.com/pytorch/pytorch/pull/24964 (cc @jerryzh168).
Maybe we can change the internal implementation to just have `PerTensorQuantizer`/`PerChannelQuantizer` with the qscheme as an input to decide if it is affine/symmetric. I haven't looked in detail, but hopefully this shouldn't break other things.